custom/icds_reports/ucr/data_sources/tech_issue_cases.json

```diff
@@ -112,36 +108,29 @@
         "is_nullable": true,
         "column_id": "district_id",
         "expression": {
-          "location_id_expression": {
-            "location_id_expression": {
-              "test": {
-                "operator": "eq",
-                "expression": {
-                  "datatype": null,
-                  "type": "property_name",
-                  "property_name": "aww_or_ls"
-                },
-                "type": "boolean_expression",
-                "property_value": "aww"
-              },
-              "expression_if_true": {
-                "location_id_expression": {
-                  "datatype": null,
-                  "type": "property_name",
-                  "property_name": "aww_id"
-                },
-                "type": "location_parent_id"
+          "location_id": {
+            "type": "conditional",
+            "test": {
+              "operator": "eq",
+              "expression": {
+                "type": "property_name",
+                "property_name": "aww_or_ls"
               },
-              "type": "conditional",
-              "expression_if_false": {
-                "datatype": "string",
+              "type": "boolean_expression",
+              "property_value": "aww"
+            },
+            "expression_if_true": {
                 "type": "property_name",
-                "property_name": "ls_id"
-              }
+                "property_name": "aww_id"
             },
-            "type": "location_parent_id"
+            "expression_if_false": {
+                "type": "property_name",
+                "property_name": "ls_id"
+            }
           },
```

Comment:
(nice to have) make this a named expression to avoid repetition

Reply:
Done